components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/model/API.java

```diff
@@ -587,4 +588,20 @@ public boolean isDefaultVersion(){
     public boolean isPublishedDefaultVersion(){
         return isPublishedDefaultVersion;
     }
+
+    public String getAllowedOrigins() {
```

Comment:
Here, the method name suggests that it returns multiple values. However, "String" is returned as the output though, the natural expectation would be to have a "String array" or something used as the return type. Any particular reason why an array is not used?